content/github/getting-started-with-github/keyboard-shortcuts.md

```diff
@@ -37,6 +37,7 @@ Below is a list of some of the available keyboard shortcuts.
 |<kbd>g</kbd> <kbd>a</kbd> | Go to the **Actions** tab. For more information, see "[About Actions](/actions/getting-started-with-github-actions/about-github-actions)."{% endif %}
 |<kbd>g</kbd> <kbd>b</kbd> | Go to the **Projects** tab. For more information, see "[About project boards](/articles/about-project-boards)."
 |<kbd>g</kbd> <kbd>w</kbd> | Go to the **Wiki** tab. For more information, see "[About wikis](/communities/documenting-your-project-with-wikis/about-wikis)."
+|<kbd>g</kbd> <kbd>g</kbd> | Go to the **Discussions** tab. For more information, see "[About discussions](/discussions/collaborating-with-your-community-using-discussions/about-discussions)."
```

Comment:
We've got a failing test! It took me a minute to figure it out, but the reason it's failing is because discussions is only available in the version of GitHub we call `free-pro-team`. So the About discussions article isn't versioned for GHES or GHEA, but this article is. You can check out the `versions` line in the metadata of the [discussions article](https://github.com/github/docs/edit/main/content/discussions/collaborating-with-your-community-using-discussions/about-discussions.md). 

That means we'll have to: 
- add versioning at the end of line 39 `{% if currentVersion == "free-pro-team@latest" %}`
- add `{% endif %}` at the end of this line
You can check out lines 36 and 37 ☝️ as an example. Ping me when you've made that change (or let me know if you have any questions!) and we'll get this merged in.

Reply:
Sounds good! 😄 Apart from viewing the metadata, is there a way to check which version of GitHub a features belong to?

Reply:
I think I resolved the issue correctly, let me know if anything needs changing. 

@janiceilene